### DISCUSSION TITLE
[iOS}: Notification Icon not Mapped

### DISCUSSION THREAD

**Discussion by SC-HeinrichG** [thudugala/Plugin.LocalNotification#502] [Ideas]
Before I start,
Excellent Plug-In and thanks for making it available.

I see in UNNotificationContent which is the content of the push Notification that [UNNotificationActionIcon](https://developer.apple.com/documentation/usernotifications/unnotificationactionicon)
is not mapped. Resulting in not being able to change the Notification Icon.
Would it be possible to map this property to be used in sending a LocalNotification?

King regards
Heinrich Glover

**Comment by thudugala** [comment]
@SC-HeinrichG try version 11.1.3
